src/nandini/Exam1.java

```diff
@@ -0,0 +1,51 @@
+package nandini;
+
+public class Exam1 {
+
+	 String getMaxLengthName(String[] arr){
+		int i =0; 
+		int length = 0;
+		int maxLengthName = arr[0].length();
+		for(int index = 1; index < arr.length; index++) {
+			
+			if(arr[index].length() > maxLengthName) {
+				i = index;
+				maxLengthName = arr[index].length();
+			}
+			    
+		}
+
+		length = arr[i].length();
+		System.out.println("length of the longest Name is :" + length);
+		
+		return arr[i];
+	}
+	 
+	 void display(String name) {
+		 
+		 String evenCharcters = "";
+		 for(int index = 1; index < name.length(); index++) {
+			  
+			if(index % 2 == 0)
```

Comment:
look at the requirement. we want to print even character , index starts from 0. Odd index represents even characters of the string. look at the expected output. 

You should write if(index%2 != 0)